.gitignore

```diff
@@ -90,7 +90,7 @@ ENV/
 /output/
 /notebooks/output/
 /notebooks/data/
-example-notebooks/binary-classifier/dictionary.yml
```

Comment:
Update notebook to save to this loc after [data!3](https://github.com/epic-open-source/seismometer-data/pull/3)

Reply:
can you merge data!3, so we can do this update?

Reply:
updated